news/2018-01-31-uioPlus.html

```diff
@@ -0,0 +1,168 @@
+                </span>
+            </div>
+
+            <!-- This is the div that will contain the Preference Editor component -->
+            <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
+
+            <!--
+                This div is for the sliding panel bar that shows and hides the Preference Editor controls in the desktop view.
+                A separate panel bar for desktop displays is needed to preserve the correct tab order.
+            -->
+            <div class="fl-panelBar fl-panelBar-wideScreen">
+                <span class="fl-prefsEditor-buttons">
+                    <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide"> Show/Hide</button>
+                    <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                </span>
+            </div>
+        </div>
+        <!-- END markup for Preference Editor -->
+
+        <div id="skip">
+            <a href="#content">Skip to Content</a>
+        </div>
+
+        <div class="floe">
+            <header class="floe-header">
+                <h1><a rel="home" title="Home" href="../index.html" class="floe-header-logo">floe
+                    <span class="floe-header-tagline">
+                        flexible learning for open education
+                    </span></a>
+                </h1>
+            </header>
+
+            <div class="flc-toc-tocContainer toc"> </div>
+            <article class="floe-content floe-news-item">
+                <h2> News </h2>
+
+                <!-- BEGIN markup for news item -->
+                <h3>Introducing UI Options Plus (UIO+)</h3>
+                <time class="floe-date" datetime="2018-01-31">January 31, 2018</time>
+                <p>
+                    User Interface Options also known as Learner Options provides individuals with the ability to customize a web
+                    site to meet their personal needs and preferences. For example, if the contrast of this page isn't quite right, you can use the tab at the top
+                    to modify how things are displayed. But what do you do when you are on a site that doesn't incorporate UI Options or
+                    other personalization methods?
+                </p>
+
+                <p>
+                    We have recently published a beta of <a href="https://chrome.google.com/webstore/detail/ui-options-plus-uio+/okenndailhmikjjfcnmolpaefecbpaek">User Interface Options Plus (UIO+)</a> to the Chrome Web Store.
+                    UIO+ is a new tool to enable the personalization experience of UI Options as a browser extension.
+                    Preferences can be set via the UIO+ adjuster panel or, if on a <a href="https://gpii.net/">GPII (Global Public Inclusive Infrastructure)</a> enabled machine, from a keyed in preference set.
+                </p>
```

Comment:
Some of this comes across a bit awkward for me. "For example, if the contrast of this page isn't quite right, you can use the tab at the top to modify how things are displayed. But what do you do when you are on a site that doesn't incorporate UI Options or other personalization methods?"

Could this be better?

"Currently to provide such customizations, the web page needs to incorporate the UI Options widget into the site's markup. However, not every site will have UI Options or site personalizations incorporated - how will a visitor with specific needs and preferences manage?

UI Options+ aims to address this issue by enabling the personalization experience of UI Options as a browser extension. Preferences can be set via the UIO+ adjuster panel or, if on a <a href="https://gpii.net/">GPII (Global Public Inclusive Infrastructure)</a> enabled machine, from a keyed in preference set.

We have recently published a beta of <a href="https://chrome.google.com/webstore/detail/ui-options-plus-uio/okenndailhmikjjfcnmolpaefecbpaek">User Interface Options Plus (UIO+)</a> to the Chrome Web Store."